client/src/client.rs

```diff
@@ -163,13 +163,14 @@ impl RawTx for String {
 }
 
 /// The different authentication methods for the client.
-pub enum Auth<'a> {
+#[derive(Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub enum Auth {
     None,
     UserPass(String, String),
-    CookieFile(&'a str),
+    CookieFile(String),
```

Comment:
Since we are talking about a file path, wouldn't `std::path::PathBuf` be a more appropriate type?

Reply:
If we want it to be owned, I think pathbuf makes sense indeed.

Reply:
Done.